blog/2022-06-23-what-we-build-right-now-june22/index.md

```diff
@@ -0,0 +1,55 @@
+---
+slug: what-we-build-right-now-june22
+title: "What we're building on the platform right now: 5GSA, 5GNSA, OCS, content filtering, VoNR, VoLTE on sXGP, VoLTE roaming, support for PBX from Telavox... and more "
+date: 2022-06-09
+tags: [saas, operators, 5g, ocs, vonr, volte, pbx]
```

Comment:
Would a "building-now" tag make sense to group all of these blog posts (assuming you keep the same format)?

Reply:
Thanks and yes, adding.